kong/core/handler.lua

```diff
@@ -128,14 +130,34 @@ return {
           "' with: "..tostring(err))
       end
 
-      -- if set `host_header` is the original header to be preserved
+      -- `uri` is the URI with which to call upstream, as returned by the
+      -- router, which might have truncated it (`strip_uri`).
+      --
+      -- If set `host_header` is the original header to be preserved.
```

Comment:
I like grouping multiple `var` or `ctx` assignments together, as is done multiple times in that function